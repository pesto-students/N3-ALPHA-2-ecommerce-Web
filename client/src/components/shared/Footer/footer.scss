@import '../../shared/Style/index';

.footer {
    background: $black-343A40;
    padding: $padding;
    margin-top: 4rem;
    &_content {
        @include sectionWidth;
    }
    &_text {
        color: $grey-bbb;
        margin: 0;
    }
}
