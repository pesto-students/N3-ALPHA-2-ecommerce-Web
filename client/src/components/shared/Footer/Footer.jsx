import React from 'react';
import './footer.scss';

const Footer = () => {
    return (
        <footer className="footer">
            <div className="footer_content">
                <p className="footer_text">
                    Â© 2021 HyGenie || All Rights Reserved.
                </p>
            </div>
        </footer>
    );
};

export default Footer;
