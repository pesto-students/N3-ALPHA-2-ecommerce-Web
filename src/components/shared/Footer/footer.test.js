import renderer from 'react-test-renderer';
import Footer from './Footer';

test('Footer Snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
});
