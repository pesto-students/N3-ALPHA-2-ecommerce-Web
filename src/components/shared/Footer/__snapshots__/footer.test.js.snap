// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="footer_content"
  >
    <p
      className="footer_text"
    >
      Â© 2021 HyGenie || All Rights Reserved.
    </p>
  </div>
</footer>
`;
